### PR TITLE
Updated tools and expanded Helm chart

### PR DESCRIPTION
**Pull request by Jaybee18**
# Description

The purpose of this pull request is mainly to improve the security of the project.

## Changes

The PR updates multiple used modules as well as the go version of the project to remove CVE findings.
It also makes sure that the docker container does not run as root and updates the Dockerfile and the Helm Chart accordingly.

## Approval for review

- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

**Comment by Jaybee18** [comment]
Hey @bischofmax @mamutmk5,

It seems that the original author of this repo, @MarkusBartels, is no longer active.
Can someone else please have a look at this pull request?

**Comment by mamutmk5** [comment]
> Hey @bischofmax @mamutmk5,
> 
> It seems that the original author of this repo, @MarkusBartels, is no longer active. Can someone else please have a look at this pull request?

Hi @Jaybee18 thank you for your pr. Yes the original author is not more active. I have assigned and inform the Team who is in charge of it.

**Comment by Jaybee18** [comment]
Thank you!

**Comment by Jaybee18** [comment]
Hello @hpi-schul-cloud/sre-team,

I would like to just check in with you and ask for a status update.
Are you currently working on this or is there maybe an estimate when you can start working on this pr?

**Comment by aimee-889** [comment]
Hello [Jaybee18](https://github.com/Jaybee18) :) Thank you much for the PR. We plan to review it, but we are a bit short on manpower at the moment. We added it to our To Do List and hope to be able to review it soon.

File: Dockerfile [hunk 1]
```diff
@@ -15,10 +15,14 @@ COPY . .
 
 RUN CGO_ENABLED=0 go build -o webhook -ldflags '-w -extldflags "-static"' .
 
-FROM alpine:3.9
+FROM alpine:3.20.0
```
Comment (dimapin) [inline Dockerfile]: For base images we define only major and minor versions to have an updated version on a new build

**Review by dimapin — CHANGES_REQUESTED** [review]
Thank you for your PR @Jaybee18. Can you please add the "properties" parameter to the README.md.

**Review by dimapin — APPROVED** [review]

**Pull request merged**